### PR TITLE
Fixes #50 Creation of appointment model

### LINKED ISSUE #50: [Feature Request]: Create Appointment Model
### Is your feature request related to a problem? Please describe.

Create Appointment Model

### Describe the solution you'd like

Refer to https://github.com/rt4914/KlinicCon-Backend/wiki/DB-Design

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by vanamraghu**
1. Added appointment migration with columns except payment_id
2. Schema is updated with details about appointment
3. Appointment model added with enum details for status

Resolves #50 

### Description

Creation of appointment model & migration

### Type of change

- [ ] Feature

### How to test this PR?

1. Check appointment column_names as below

irb(main):002> Appointment.column_names
=> 
["id",
 "patient_profile_id",
 "service_id",
 "slot_start_time",
 "slot_end_time",
 "status",
 "created_at",
 "updated_at"]

### CODE REVIEW

File: app/models/appointment.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+class Appointment < ApplicationRecord
+  belongs_to :patient_profile
+  belongs_to :service
+  enum status: { NOT_CONFIRMED: 0, CONFIRMED: 1, CANCELLED: 2, COMPLETED: 3 }
```
Comment (rt4914) [inline app/models/appointment.rb:5]: COMPLETED should be 2 and CANCELLED should be 3

File: app/models/appointment.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+end
+
```
Comment (rt4914) [inline app/models/appointment.rb:7]: Keep only 1 blank line at the end.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@vanamraghu Minor change.

File: app/models/appointment.rb (hunk 2 above)
Comment (vanamraghu) [reply app/models/appointment.rb:7]: Done

File: app/models/appointment.rb (hunk 1 above)
Comment (vanamraghu) [reply app/models/appointment.rb:5]: Fixed

**Review by rt4914 — CHANGES_REQUESTED** [review]
This PR introduces changes to Gemfile and Gemfile.lock, please remove those changes.

**Comment by vanamraghu** [comment]
> This PR introduces changes to Gemfile and Gemfile.lock, please remove those changes.

@rt4914 
Removed the gem file

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**